integration_tests/mockApis/supportAdditionalNeedsApi/challengesEndpoints.ts

```diff
@@ -85,7 +85,7 @@ const stubGetChallenges = (
             updatedByDisplayName: 'Alex Smith',
             updatedAt: '2023-06-19T09:39:44Z',
             updatedAtPrison: 'MDI',
-            symptoms: 'John is great at adding up',
+            symptoms: 'John is not good at long division',
```

Comment:
Minor change; just making the stub data for the challenges endpoint indicative of a challenge